cms/envs/aws.py

```diff
@@ -110,6 +110,12 @@
 # DEFAULT_COURSE_ABOUT_IMAGE_URL specifies the default image to show for courses that don't provide one
 DEFAULT_COURSE_ABOUT_IMAGE_URL = ENV_TOKENS.get('DEFAULT_COURSE_ABOUT_IMAGE_URL', DEFAULT_COURSE_ABOUT_IMAGE_URL)
 
+# DEFAULT_COURSE_VISIBILITY_IN_CATALOG specifies the default catalog visibility of a course when it is created
```

Comment:
This comment seems unnecessary given the verbose name of the variable.  I suggest removing the comment altogether.